src/website/app/demos/Icon/examples/icon.js

```diff
@@ -20,6 +20,11 @@ import IconSentimentSatisfied from '../../../../../Icon/IconSentimentSatisfied';
 export default {
   id: 'icon',
   title: 'Icon',
+  description: `Bundled Icons should be imported directly
+
+\`import IconSentimentSatisfied from 'mineral-ui/Icon/IconSentimentSatisfied'\`.
```

Comment:
Maybe this is better presented as a commented-out line in the source? As it is, the wrapping makes it really hard to parse. We should apply the same treatment to the custom example, too.

Reply:
I just shortened it so the line wouldn't wrap. We spent so much time on import paths for icons, I wanted to call this out :)

Reply:
👌